lib/src/base-styles/_nx-lists.scss

```diff
@@ -49,32 +49,42 @@
   }
 
   > .nx-loading-spinner {
-    -ms-grid-row: 1;
-    -ms-grid-column: 2;
     grid-area: list-text;
     margin: 0;
     text-align: center;
   }
 }
 
+.nx-list__item--link {
+  display: contents;
+}
+
+.nx-list__link {
+  text-decoration: none;
+  color: inherit;
+}
+
 .nx-list__text {
   @include semi-bold();
   @include container-horizontal;
 
   display: block;
-  -ms-grid-row: 1;
-  -ms-grid-column: 2;
   grid-area: list-text;
   margin: 0;
   padding-right: $nx-spacing-l;
+
+  &.nx-truncate-ellipsis {
+    // Something about the grid layout gets confused due to overflow: hidden on this element
+    //  and adds more vertical space to the grid, in the second row.
+    // Experimentally, setting an explicit height here fixes that.
```

Comment:
```suggestion
    // Setting an explicit height here fixes that.
```

Reply:
> // Setting an explicit height here fixes that.

I disagree with this one.  Pointing out that we don't know why this worked is valuable.